### PR TITLE
Changelog for release 46.9.0

### PR DESCRIPTION
**Pull request by praksb**
### What does this PR do?
Update changelog for relase 46.9.0

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
+## 46.9.0 - August 8, 2019
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Process errors not in JSON format to make them user friendly before displaying ([PR #1521](https://github.com/forcedotcom/salesforcedx-vscode/pull/1521))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md]: This PR was released in 46.8.0 so we can remove it.

Comment (praksb) [reply packages/salesforcedx-vscode/CHANGELOG.md]: will do.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
+## 46.9.0 - August 8, 2019
+
+## Added
+
+#### salesforcedx-utils-vscode
+
+- Use VS Code api to open browser in host machine when operating in container mode ([PR #1448](https://github.com/forcedotcom/salesforcedx-vscode/pull/1448))
```
Comment (hbc-bindu) [inline packages/salesforcedx-vscode/CHANGELOG.md]: @praksb, while reviewing a different message Nate mentioned that container refers to a very specific situation and he would prefer headless instead. Let me know if this looks ok:
Use VC Code API to open a browser in the host machine when operating in headless environment.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+
+#### docs
+
+- Add Japanese translations for docs ([PR #1528](https://github.com/forcedotcom/salesforcedx-vscode/pull/1528))
```
Comment (hbc-bindu) [inline packages/salesforcedx-vscode/CHANGELOG.md]: @praksb, Do we have a location where the Japanese version of the docs will be uploaded? In which case, could we include that link here?
Include Japanese translation for Salesforce Extensions for VS Code documentation <add link>

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 2 above)
Comment (praksb) [reply packages/salesforcedx-vscode/CHANGELOG.md]: that sounds good to me. i'll update the comment.

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 3 above)
Comment (praksb) [reply packages/salesforcedx-vscode/CHANGELOG.md]: I think this is the link. let me add it.
https://github.com/forcedotcom/salesforcedx-vscode/tree/develop/docs/articles-jp

**Pull request merged**